bundles/org.openhab.binding.senechome/src/main/java/org/openhab/binding/senechome/internal/SenecSystemStatus.java

```diff
@@ -113,10 +113,11 @@ public enum SenecSystemStatus {
     SPARE_CAPACITY(89, "SPARE CAPACITY"),
     SELFTEST_ERROR(90, "SELFTEST ERROR"),
     EARTH_FAULT(91, "EARTH FAULT"),
-    PV_MODE(92, "PV MODE"),
+    PVMODE(92, "PV-MODE"),
```

Comment:
There is no code behind that has to be adapted?

Reply:
I just posted to the comment the latest .js file i downloaded from my Senechome device. There it is named `92: "PV-MODE"`. I don't know if it was a typo before. I just synced it with the latest version.